webapp/sass/common.scss

```diff
@@ -80,7 +80,7 @@ tt {
     }
 
     span {
-      text-shadow: 0 0 1px var(--ps-neutral-1);
```

Comment:
@eh-am so apparently this affects the shadow on the tables... Why is this css located here instead of in `packages/pyroscope-flamegraph/src/ProfilerTable.module.scss` ? same with all other table related css code?